src/components/chatRoom/ChatBoxItem.tsx

```diff
@@ -0,0 +1,94 @@
+import { useRecoilValue } from 'recoil';
+import styled from 'styled-components';
+
+import { curUserState } from '../../recoil/recoil';
+import { User } from '../../interfaces/interfaces';
+
+const ChatBox = ({
+  user,
+  chat_content,
+}: {
+  user: User;
+  chat_content: string;
+}) => {
+  const curUser = useRecoilValue(curUserState);
+
+  return (
+    <ChatBoxItem isSelected={user.user_id === curUser}>
+      {user.user_id === curUser ? (
+        <></>
+      ) : (
+        <UserImgWrapper>
+          <UserImg src={user.user_img} />
+        </UserImgWrapper>
+      )}
+      <ChatWrapper>
+        {user.user_id === curUser ? (
+          <></>
+        ) : (
+          <ChatUserName>{user.user_name}</ChatUserName>
+        )}
+        <ChatItem>
+          <ChatText isSelected={user.user_id === curUser}>
+            {chat_content}
+          </ChatText>
+          <ChatTime />
+        </ChatItem>
+      </ChatWrapper>
+    </ChatBoxItem>
+  );
+};
+
+const ChatBoxItem = styled.div<{ isSelected: boolean }>`
+  display: flex;
+  gap: 0.8rem;
+
+  justify-content: ${(props) => (props.isSelected ? 'right' : 'left')};
+`;
+
+const UserImgWrapper = styled.div`
+  width: 2.2rem;
+  height: 2.2rem;
+  border-radius: 40%;
+  overflow: hidden;
+`;
+
+const UserImg = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
+
+const ChatWrapper = styled.div``;
+
+const ChatUserName = styled.div`
+  padding-bottom: 0.4rem;
+
+  font-size: 0.75rem;
+`;
+
+const ChatItem = styled.div`
+  display: flex;
+  gap: 0.5rem;
+`;
+
+const ChatText = styled.div<{ isSelected: boolean }>`
+  padding: 0.6rem 0.8rem 0.5rem;
+
+  border-radius: 5px;
+
+  font-size: 0.9rem;
+  font-weight: 500;
+
```

Comment:
```suggestion
  word-break: break-all;
```
![image](https://user-images.githubusercontent.com/103591752/200160363-cc0117b1-838a-4a65-995a-2035e5b2ff09.png)
![image](https://user-images.githubusercontent.com/103591752/200160392-9a32bab8-13f4-4d62-96a8-e87e6eed7b00.png)

채림님한테 배운 건데... 이렇게 하면 영어로 길게 쳤을 때도 말풍선이 깨지지 않아용!!